common/queue/src/main/java/org/thingsboard/server/queue/pubsub/TbPubSubProducerTemplate.java

```diff
@@ -120,7 +121,10 @@ private Publisher getOrCreatePublisher(String topic) {
             try {
                 admin.createTopicIfNotExists(topic);
                 ProjectTopicName topicName = ProjectTopicName.of(pubSubSettings.getProjectId(), topic);
-                Publisher publisher = Publisher.newBuilder(topicName).setCredentialsProvider(pubSubSettings.getCredentialsProvider()).build();
+                Publisher publisher = Publisher.newBuilder(topicName)
+                        .setCredentialsProvider(pubSubSettings.getCredentialsProvider())
+                        .setExecutorProvider(FixedExecutorProvider.create(pubSubSettings.getTbPubSubQueueExecutorProvider().getExecutor()))
```

Comment:
This change is not consistent with the way how we set the CredentialsProvider. PubSubSettings already has the `getCredentialsProvider()` method but does not have the `getExecutorProvider()`